repairnator/repairnator-pipeline/src/main/java/fr/inria/spirals/repairnator/pipeline/Launcher.java

```diff
@@ -126,6 +128,14 @@ private JSAP defineArgs() throws JSAPException {
         opt2.setHelp("Specify the build id to use.");
         jsap.registerParameter(opt2);
 
+        opt2 = new FlaggedOption("nextBuild");
+        opt2.setShortFlag('n');
+        opt2.setLongFlag("nextBuild");
+        opt2.setStringParser(JSAP.INTEGER_PARSER);
+        opt2.setDefault("0");
```

Comment:
default value should be NO_PATCH after your refactoring above

Reply:
Done.